.helix/languages.toml

```diff
@@ -49,4 +49,4 @@ auto-format = true
 # This contains words that I know are spelled correctly,
 # so ltex-ls doesn't give me a warning
 [language-server.ltex-ls.config.ltex.dictionary]
-"en-US" = [] 
+"en-US" = ["pnpm"] 
```

Comment:
This doesn't actually do what I want it to do, so I'll have to look at this later to see what's going on. I should comment that.